app/lib/three_scale/collection_representer.rb

```diff
@@ -21,4 +21,13 @@ def to_xml(options = {})
       xml_collection.new(self).to_xml(options.merge(root: representation_wrap))
     end
   end
+
+  module OrderedCollection
+    %i[xml json].each do |format|
+      define_method "to_#{format}" do |*args, &block|
+        replace(self.sort_by(&:id)) if System::Database.postgres? && respond_to?(:replace)
```

Comment:
It should be just `replace(self.sort_by(&:id)) if respond_to?(:replace)`
or even simpler:`try(:replace, sort_by(&:id))`

Because either we order by all databases or we don't guarantee any order 😄 But I guess it is a good idea to order, specially for pagination 😜